core/src/utils/native/haptic.ts

```diff
@@ -28,37 +94,34 @@ const HapticEngine = {
      * the Vibrate API. This check avoids that error
      * if the browser does not support the Vibrate API.
      */
-    if (win.Capacitor?.getPlatform() === 'web') {
+    if (capacitor?.getPlatform() === 'web') {
       return typeof navigator !== 'undefined' && navigator.vibrate !== undefined;
     }
 
     return true;
   },
   isCordova() {
-    return !!(window as any).TapticEngine;
+    return (window as any).TapticEngine !== undefined;
```

Comment:
Are we confident that `TapticEngine` is undefined when unset? (as opposed to null or `''`). Is this documented somewhere?